test/integration/cloud-config-aws.yml.template

```diff
@@ -14,6 +14,8 @@
 aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
+aws_region: @REGION
```

Comment:
All of my tests (so far) seem to be working fine on us-east-2. I'm specifying us-east-2 particularly to identify any obvious flaws assuming us-east-1. That's why I need this extra variable.